src/EditorFeatures/CSharpTest2/Recommendations/VarKeywordRecommenderTests.cs

```diff
@@ -251,5 +251,22 @@ public VarKeywordRecommenderTests()
 @"class C {
      void M1(out $$");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestVarPatternInSwitch()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(
+@"switch(o)
+    {
+        case $$
```

Comment:
See the "var pattern" in the [pattern spec](https://github.com/dotnet/csharplang/blob/master/proposals/patterns.md#var-pattern). I didn't know about this until I saw the talk ;-)
I think the main usage would be in conjunction with `when`. Also, it seems good for completeness (you can use `var` in places where you can use a type).